### PR TITLE
Small adjustments in steps after follow it

### PR DESCRIPTION
**Pull request by camilamacedo86**

### CODE REVIEW

**Comment by camilamacedo86** [comment]
@nialldonnellyfh Done:-)

File: Getting-Started.md [hunk 1]
```diff
@@ -132,7 +131,15 @@ RHMAP provides a set of Raincatcher project templates to help you get the Rainca
 
   3. If it does not indicate success, click **Deploy Cloud App** and verify the next deployment is successful.
 
-The apps are now created, configured, and deployed.
+  ** The apps are now created, configured, and deployed. **
+
+10. Check the web app deploment in RHMAP
```
Comment (pmdarrow) [inline Getting-Started.md]: *deployment

File: Getting-Started.md [hunk 2]
```diff
@@ -132,7 +131,15 @@ RHMAP provides a set of Raincatcher project templates to help you get the Rainca
...
+  1. Select the **Projects** header menu item.
+  2. Select the **wfm-demo** project.
+  3. Select the web app.
+  4. From the left-hand side menu, select **Details**.
+  5. Click **Current Hoste**.
```
Comment (pmdarrow) [inline Getting-Started.md]: *Host

**Comment by camilamacedo86** [comment]
Done corrections @pmdarrow . Tks 👍

**Pull request merged**